PhysEdJournal.Api/Rest/Common/Response.cs

```diff
@@ -0,0 +1,26 @@
+using PResult;
+
+namespace PhysEdJournal.Api.Rest.Common;
+
+public static class Response
+{
+    public static IResult Ok<T>(T data)
+    {
+        return Results.Ok(new { success = "success", data });
+    }
+
+    public static IResult Ok(Unit data)
+    {
+        return Results.Ok(new { success = "success", data = (Unit?)null });
```

Comment:
Здесь было бы неплохо оставить коммент зачем мы это вообще делаем

```suggestion
        // In case of no data we need to return json with
        // success: true, data: null. To do this we cast null to 
        // any nullable type
        return Results.Ok(new { success = "success", data = (Unit?)null });
```